test/src/tests-persistence.ts

```diff
@@ -1661,6 +1661,53 @@ describe('persistence', () => {
         parsedProductsAfter['testMPID'].cp.length.should.equal(0);
     });
 
+    describe('products persistence with noTargeting privacy flag', () => {
+        beforeEach(() => {
+            mParticle._resetForTests(MPConfig);
+        }); 
+
+        it('should not save products LS when noTargeting is true', async () => {
+            mParticle.config.noTargeting = true;
```

Comment:
I'm surprised this is working.  

Per the [docs](https://docs.rokt.com/developers/integration-guides/getting-started/ecommerce/ecommerce-sdk-integration#overview), this should be:
```suggestion
            mParticle.config.launcherOptions = {noTargeting: true};
```